common/config/rush/pnpm-lock.yaml

```diff
@@ -2839,7 +2995,7 @@ packages:
       integrity: sha512-bC7ElrdJaJnPbAP+1EotYvqZsb3ecl5wi6Bfi6BJTUcNowp6cvspg0jXznRTKDjm/E7AdgFBVeAPVMNcKGsHMA==
   /debug/3.2.6:
     dependencies:
-      ms: 2.1.3
+      ms: 2.1.1
```

Comment:
yet another downgraded

Reply:
I think the issue with the package downgrades is our `pnpm-lock.yaml` contains multiple versions of the same package which match the requested version spec, and this seems to trigger non-deterministic behavior in rush+pnpm.

For `ms`, our `pnpm-lock.yaml` contains both `2.1.3` and `2.1.1`.  `2.1.3` is the latest, but `mocha@7.2.0` pins to `2.1.1`:

https://github.com/mochajs/mocha/blob/v7.2.0/package.json#L68

If we upgraded `mocha` to `^8.3.0` I believe this problem would go away:

https://github.com/mochajs/mocha/blob/v8.3.0/package.json#L69

Reply:
Looking at this further, I think the new behavior may actually be more correct.  `debug@3.2.6` is only used by `mocha@7.2.0`, which pins `ms` to `2.1.1`.  So it seems more correct for `debug@3.2.6` to depend on `ms@2.1.1` rather than `ms@2.1.3`.  I don't know whether this is an intentional improvement in rush/pnpm, or if it's just unspecified behavior that happened to flip to the "more correct" version this run.